assets/dev/js/frontend/handlers/section.js

```diff
@@ -1,3 +1,155 @@
+class BackgroundSlideshow extends elementorModules.frontend.handlers.Base {
+	getDefaultSettings() {
+		return {
+			selectors: {
+				swiperContainer: '.elementor-section-background-slideshow',
+				swiperSlides: '.swiper-slide',
+			},
+			classes: {
+				swiperContainer: 'elementor-section-background-slideshow swiper-container',
+				swiperWrapper: 'swiper-wrapper',
+				swiperSlide: 'elementor-section-background-slideshow__slide swiper-slide',
+				swiperSlideInner: 'elementor-section-background-slideshow__slide__image',
+			},
+		};
+	}
+
+	getSwiperOptions() {
+		const elementSettings = this.getElementSettings();
+
+		const swiperOptions = {
+			grabCursor: false,
+			slidesPerView: 1,
+			slidesPerGroup: 1,
+			loop: 'yes' === elementSettings.background_slideshow_loop,
+			speed: elementSettings.background_slideshow_animation_speed,
+			autoplay: {
+				delay: elementSettings.background_slideshow_autoplay_speed,
+				disableOnInteraction: false,
+				stopOnLastSlide: 'yes' !== elementSettings.background_slideshow_loop,
+			},
+		};
+
+		if ( 'yes' === elementSettings.background_slideshow_loop ) {
+			swiperOptions.loopedSlides = this.getSlidesCount();
+		}
+
+		switch ( elementSettings.background_slideshow_content_animation ) {
+			case 'fade':
+				swiperOptions.effect = 'fade';
+				break;
+			case 'slide_right':
+				swiperOptions.initialSlide = this.getSlidesCount() - 1;
+				swiperOptions.autoplay.reverseDirection = true;
+				break;
+			case 'slide_left':
+				break;
+			case 'slide_up':
```

Comment:
You can swipe between slide up and down, then avoid `swiperOptions.direction = 'vertical'` duplication